### ISSUE TITLE
Add translations and get transifex set up

### ISSUE THREAD

**Issue by HenrikJannsen** [bisq-network/bisq2#1391]
Before starting to use transifex it would be good to clean up and optimize organization of the translation strings in bisq2.
See https://github.com/bisq-network/bisq2/issues/1392

**Comment by HenrikJannsen** [comment]
@https://github.com/hiciefte Would you be interested to work on that?

**Comment by hiciefte** [comment]
> https://github.com/hiciefte Would you be interested to work on that?

Yes, please also assign me for this task! 👍

**Comment by hiciefte** [comment]
@mwithm: Is it you, I need to contact to create a new transfex project and give me the necessary permissions for now or is it @ripcurlx I should contact?

**Comment by hiciefte** [comment]
I looked into the documentation and found a GitHub integration that would be one way to do it (https://github.com/apps/transifex-integration), but it would grant writing access to the repo. It could be configured to do PRs instead of commits, but still that it just a configuration option. I guess the safer way is to do it as I saw it in Bisq to access it via the API and update it with a script (https://github.com/bisq-network/bisq/blob/bba732711592210d3481edcc137692004bd89d63/core/update_translations.sh). WDYT?
